demos/python_demos/speech_recognition_demo/README.md

```diff
@@ -16,20 +18,20 @@ After computing audio features, running a neural network to get per-frame charac
 
 The demo depends on the `ctcdecode_numpy` Python extension module,
 which implements CTC decoding in C++ for faster decoding.
-Please refer to [Using Open Model Zoo demos](../../README.md) for instructions
+Please refer to [Open Model Zoo demos](../../README.md#build-the-demo-applications) for instructions
 on how to build the extension module and prepare the environment for running the demo.
+Alternatively, instead of using `cmake` you can run `python -m pip install .` inside `ctcdecode-numpy` directory to build and install `ctcdecode-numpy`.
 
 ## Model preparation
 
-You can download and convert a pre-trained Mozilla\* DeepSpeech 0.6.1 model with
-OpenVINO [Model Downloader](../../../tools/downloader/README.md).
-This essentially boils down to the following commands:
+You can download and convert a pre-trained Mozilla\* DeepSpeech 0.8.2 or 0.6.1 model with
+OpenVINO [Model Downloader](../../../tools/downloader/README.md) and the provided conversion scripts.
+This is done with the following commands:
```

Comment:
You say that you can download/convert 0.8.2 or 0.6.1 with the following commands, but the commands are only for 0.8.2.

Reply:
Done: added commands for 0.6.1 as well.  When I was writing this, I assumed that would be obvious for the users.